lib/app/features/feed/content_notification/views/components/content_notification_bar.dart

```diff
@@ -1,55 +0,0 @@
-// SPDX-License-Identifier: ice License 1.0
-
-import 'package:flutter/material.dart';
-import 'package:flutter_animate/flutter_animate.dart';
-import 'package:hooks_riverpod/hooks_riverpod.dart';
-import 'package:ion/app/extensions/extensions.dart';
-import 'package:ion/app/features/feed/content_notification/data/models/content_notification_data.c.dart';
-import 'package:ion/app/features/feed/content_notification/providers/content_notification_provider.c.dart';
-import 'package:ion/app/features/feed/stories/effects/slide_in_out_effects.dart';
-
-class ContentNotificationBar extends ConsumerWidget {
-  const ContentNotificationBar({
-    required this.data,
-    required this.height,
-    super.key,
-  });
-
-  final NotificationData data;
-  final double height;
-
-  @override
-  Widget build(BuildContext context, WidgetRef ref) {
-    return SafeArea(
-      child: Animate(
-        key: ValueKey(data),
-        effects: topSlideInOutEffects,
```

Comment:
Let's remove `topSlideInOutEffects` as well if it is unused anymore

Reply:
Good catch 🎣, removed